ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/Benchmark.scala

```diff
@@ -117,4 +128,12 @@ object Benchmark {
           Left("Metrics objectives not met.")
         else Right(())
       }
+
+  def scheduleCancelStreamTask(timeoutInSeconds: Long, observer: ObserverWithResult[_, _])(implicit
+      ec: ExecutionContext
+  ): Unit = {
+    val _ = Delayed.by(t = Duration(timeoutInSeconds, TimeUnit.SECONDS))(
```

Comment:
Yeah, had to add `val _ = ...` because the compiler complained.
I guess it doesn't really matter if we use that or `discard` or (possibly) type ascription `expr: Unit`